include/swift/AST/DiagnosticsSema.def

```diff
@@ -5579,5 +5579,36 @@ ERROR(atomics_ordering_must_be_constant, none,
       "ordering argument must be a static method or property of %0",
       (Identifier))
 
+//------------------------------------------------------------------------------
+// MARK: access notes
+//------------------------------------------------------------------------------
+
+WARNING(attr_added_by_access_note, none,
```

Comment:
These probably ought to be remarks, not warnings, to prevent -warnings-as-errors from kicking in.